### ISSUE TITLE
Accessibility docs.

### ISSUE THREAD

**Issue by Aladdin-ADD** [eslint/eslint.github.io#368]
Web accessibility (also referred to as a11y) is the design and creation of websites that can be used by everyone. Accessibility support is necessary to allow assistive technology to interpret web pages.

linking http://a11yproject.com/

**Comment by ilyavolodin** [comment]
Anything particular that you think should be fixed for better accessibility? From what I can tell, our site is at least AA compliant. The only issues I'm seeing are contrast issues on the documentation pages, and maybe a few unclear links.

**Comment by Aladdin-ADD** [comment]
thanks for clarification. what about adding the docs like this: https://facebook.github.io/react/docs/accessibility.html

**Comment by ilyavolodin** [comment]
React is a front-end framework, and that document describes how to write react code so that output web-page will be accessible. Since ESLint doesn't output any code to the web, I don't think we can create something like that.
